src/test/kotlin/no/nav/medlemskap/cucumber/steps/pdl/PdlMapperSteps.kt

```diff
@@ -173,6 +173,7 @@ class PdlMapperSteps : No {
         var sivilstander: List<HentPerson.Sivilstand> = emptyList()
         var familierelasjoner: List<HentPerson.ForelderBarnRelasjon> = emptyList()
         var doedsfall: List<HentPerson.Doedsfall> = emptyList()
+        var adressebeskyttelse: List<HentPerson.Adressebeskyttelse> = emptyList()
```

Comment:
Ser adressebeskyttelse fra PDL er modellert som en liste. Hvordan skal dette tolkes? Hvordan skal en liste med adressebeskyttelse tolkes? Finnes det noe mer info på adressebeskyttelse som kanskje burde vært hentet ut, f.eks. periode?

Reply:
Ja, jeg hentet noe jeg bruker fra soknaddialogen og så lenge en bruker har noe annet enn UGRADERT (dette er en kode som er det samme som null) i listen vil vi sende feilmelding. Vi spør også ikke om historikk, så dette er da gjeldende data for bruker. Det kan hende bruker har flere koder som er gjeldende, men dette vil uansett være valideringsfeil så lenge koden ikke er UGRADERT.

Reply:
Den er grei! :)